### PR TITLE
Add Exhibition Title facet to Public Browser

### PR DESCRIPTION
**Pull request by uneebsaleem**

### CODE REVIEW

File: src/config/default.js [hunk 1]
```diff
@@ -222,6 +222,19 @@ export default {
           },
         }),
       },
+      exhibitionTitle: {
+        field: 'collectionspace_denorm:exhibition.title',
+        messages: defineMessages({
+          label: {
+            id: 'filter.title.label',
```
Comment (ray-lee) [inline src/config/default.js]: The convention is to use the filter name in the message id, so this should be `filter.exhibitionTitle.label`.

File: src/config/default.js [hunk 2]
```diff
@@ -222,6 +222,19 @@ export default {
...
+            defaultMessage: 'exhibitionTitle',
+          },
+          shortLabel: {
+            id: 'filter.title.shortLabel',
```
Comment (ray-lee) [inline src/config/default.js]: The convention is to use the filter name in the message id, so this should be `filter.exhibitionTitle. shortLabel`.

File: src/config/default.js [hunk 3]
```diff
@@ -222,6 +222,19 @@ export default {
           },
         }),
       },
+      exhibitionTitle: {
+        field: 'collectionspace_denorm:exhibition.title',
+        messages: defineMessages({
+          label: {
+            id: 'filter.title.label',
+            defaultMessage: 'exhibitionTitle',
```
Comment (ray-lee) [inline src/config/default.js]: The message should be human readable. Please make it `Exhibition title`.

File: src/config/default.js [hunk 4]
```diff
@@ -275,6 +288,7 @@ export default {
         fields: [
           'responsibleDepartment',
           'objectName',
+          'exhibitionTitle'
```
Comment (ray-lee) [inline src/config/default.js]: The message should be human readable. Please make it `Exhibition title`.

**Review by ray-lee — CHANGES_REQUESTED** [review]
A few changes needed. Thanks!

**Comment by uneebsaleem** [comment]
@ray-lee fixed and pushed!

**Comment by ray-lee** [comment]
Thanks!

**Pull request merged**